median/examples/hello_dsp/src/lib.rs

```diff
@@ -23,8 +23,9 @@ median::external! {
     impl MSPObjWrapped<HelloDSP> for HelloDSP {
         //create some signal i/o
         fn new(builder: &mut dyn MSPWrappedBuilder<Self>) -> Self {
-            builder.add_signal_inlets(2);
-            builder.add_signal_outlets(2);
+            builder.add_signal_inlets_with_assist(&[&"in left", &"in right"]);
```

Comment:
Clippy points out a needless borrow - could just be `(&["in left", "in right"])`